ateam_ui/src/src/overlay.ts

```diff
@@ -1,6 +1,7 @@
 import { RenderConfig } from "@/state"
 import { Field } from "@/field"
-import * as PIXI from "pixi.js";
+import * as PIXI from "pixi.js"
+import { Texture } from "pixi.js"
```

Comment:
Is there a reason for this import? It seems like all the code still uses PIXI.Texture so it should be covered by the line above this one

Reply:
Nope just need to remove that